### DISCUSSION TITLE
[Data display] Charts

### DISCUSSION THREAD

**Discussion by Tomperez98** [saadeghi/daisyui#2766] [Ideas / feature requests]
It would be great to have a chart component for data display

- Flowbite: https://flowbite.com/docs/plugins/charts/
- TWElements: https://tw-elements.com/docs/standard/data/charts/

**Comment by saadeghi** [comment]
Flowbite examples are using [ApexCharts](https://apexcharts.com/) and TWElements is using [Chart.js](https://www.chartjs.org/).  
You can directly use either of these libraries (or any other chart library) with daisyUI colors.

Let me know if you have a question.
